cask-cli.el

```diff
@@ -224,6 +224,20 @@ directory called bin in the root directory.
 The output is formatted as a colon path."
   (princ (concat (s-join path-separator (cask-exec-path (cask-cli--bundle))) "\n")))
 
+(defmacro cask-cli/with-package-path (&rest body)
```

Comment:
We only use the slash functions for commands/options. Can you rename this to: `cask-cli--with-package-path `